Packs/MicrosoftDefenderAdvancedThreatProtection/ParsingRules/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.yml

```diff
@@ -1,6 +1,6 @@
 name: MicrosoftDefenderAdvancedThreatProtection Parsing Rule
 id: MicrosoftDefenderAdvancedThreatProtection Parsing Rule
 fromversion: 6.10.0
-tags: []
+tags: ['test_tag']
```

Comment:
please don't forget to remove before merging :)

Reply:
no worries, enough release notes validations are preventing me from merging this :)